### PR TITLE
Added remove_pub_address + clarified * processing

### PR DESCRIPTION
**Pull request by pawmmm**

### CODE REVIEW

File: fip-0018.md [hunk 1]
```diff
@@ -93,9 +93,48 @@ No changes
...
+  ],
+  "max_fee": 0,
+  "tpid": "rewards@wallet",
+  "actor": "aftyershcu22"
+}
+```
+#### Processing
+When * is specified as token_code only that entry is removed. For example is FIO Address is mapped to ETH-USDC and ETH-* and user removes ETH-* as a mapping, ETH-USDC is retained.
+#### Exception handling
+Specifying * as token code will now be allowed.
```
Comment (lukestokes) [inline fip-0018.md:125]: Confused by this as the previous comment says "When * is specific as a token_code"

File: fip-0018.md [hunk 2]
```diff
@@ -93,9 +93,48 @@ No changes
 ##### Example
 No changes
 
+### Remove Public Address
```
Comment (lukestokes) [inline fip-0018.md:96]: Seems a little strange to have this named singularly but it expects and array. Is this a similar standard we've followed elsewhere? Some frameworks (like JQuery) assume most things are arrays. Are we taking a similar approach or has it been thought through as to what will be arrays and what will be single values?

**Review by lukestokes — APPROVED** [review]
Left some comments but seems okay to me in general.

**Review by dapixcasey — APPROVED** [review]

File: fip-0018.md (hunk 1 above)
Comment (pawmmm) [reply fip-0018.md:125]: Which part is confusing?

File: fip-0018.md (hunk 2 above)
Comment (pawmmm) [reply fip-0018.md:96]: This call was added in FIP-4 and it was made to be consistent with add_pub_address

**Pull request merged**